src/Microsoft.Data.Analysis/DataFrame.cs

```diff
@@ -511,7 +511,15 @@ public DataFrame Append(IEnumerable<object> row = null, bool inPlace = false)
                     }
                     if (value != null)
                     {
-                        value = Convert.ChangeType(value, column.DataType);
+                        try
+                        {
+                            value = Convert.ChangeType(value, column.DataType);
+                        }
+                        catch (Exception ex)
```

Comment:
This should be catching `FormatException`. Or rather, maybe `Append`'s called should catch this exception

Reply:
Alright, I'm going to rip this try-catch out of this PR. We can add it later if we want to. I'm more interested in getting the DateTime support in